IntegrationTests/tests_01_http/defines.sh

```diff
@@ -58,7 +58,7 @@ function do_netstat() {
 }
 
 function create_token() {
-    mktemp "$tmp/server_token_XXXXXX"
+    mktemp "${tmp:?}/server_token_XXXXXX"
```

Comment:
If you're going to use `${foo?}` can you give a helpful message on the end of the parameter expansion, e.g. `${foo?"foo is not set"}`.